documentation/docs/proptest/genslist.md

```diff
@@ -14,30 +14,62 @@ We also provide generators for [Arrow](arrow.md) as a separate module.
 
 | Generator    | Description | JVM | JS | Native |
 | -------- | ----------- | --- | --- | ------ |
-| **Numerics** |
-| `Arb.int(range)` | Randomly chosen ints in the given range. If the range is not specified then all integers are considered. The edgecases are `Int.MIN_VALUE`, `Int.MAX_VALUE`, 0, 1, -1 | ✓ | ✓ | ✓ |
-| `Arb.long(range)` | Randomly chosen longs in the given range. If the range is not specified then all longs are considered. The edgecases are `Long.MIN_VALUE`, `Long.MAX_VALUE`, 0, 1, -1 | ✓ | ✓ | ✓ |
-| `Arb.nats(range)` | Randomly chosen natural numbers in the given range. If range is not specified then the default is `Int.MAX_VALUE`. The edgecases are `Int.MAX_VALUE`, 1 | ✓ | ✓ | ✓ |
-| `Arb.negativeInts(range)` | Randomly chosen negative integers in the given range. The edgecases are `Int.MIN_VALUE`, -1 | ✓ | ✓ | ✓ |
-| `Arb.positiveInts(range)` | Randomly chosen positive integers in the given range. The edgecases are `Int.MAX_VALUE`, 1 | ✓ | ✓ | ✓ |
-| `Arb.double(range)` | Randomly chosen doubles in the given range. The edgecases are `Double.MIN_VALUE`, `Double.MAX_VALUE`, `Double.NEGATIVE_INFINITY`, `Double.NaN`, `Double.POSITIVE_INFINITY`, 0.0, 1.0, -1.0, 1e300 | ✓ | ✓ | ✓ |
-| `Arb.positiveDoubles(range)` | Randomly chosen positive doubles in the given range. The edgecases are `Double.MIN_VALUE`, `Double.MAX_VALUE`, `Double.POSITIVE_INFINITY`, 1.0, 1e300 | ✓ | ✓ | ✓ |
-| `Arb.negativeDoubles(range)` | Randomly chosen negative doubles in the given range. The edgecases are `Double.NEGATIVE_INFINITY`, -1.0 | ✓ | ✓ | ✓ |
-| `Exhaustive.ints(range)` | Returns all ints in the given range. | ✓ | ✓ | ✓ |
-| `Exhaustive.longs(range)` | Returns all longs in the given range. | ✓ | ✓ | ✓ |
-| `Arb.multiples(k, max)` | Generates multiples of k up a max value. The edgecases are `0`. | ✓ | ✓ | ✓ |
-| `Arb.factors(k)` | Generates factors of k. | ✓ | ✓ | ✓ |
 | **Nulls** |
 | `arb.orNull()` | Generates random values from the arb instance, with null values mixed in. For example, `Arb.int().orNull()` could generate `1, -1, null, 8, 17`, and so on. Has overloaded versions to control the frequency of nulls being generated.| ✓ | ✓ | ✓ |
 | `arb.orNull(nullProbability)` | Generates random values from the arb instance, with null values mixed in using the defined probability. | ✓ | ✓ | ✓ |
-| **Bytes** |
-| `Arb.byte(min, max)` | Generates bytes within the given bounds. The edgecases are `0`, `-1`, `1` `min`, `max`. | ✓ | ✓ | ✓ |
-| `Arb.byteArrays(generateArrayLength, max)` | Generates byte arrays with the size of the array taken as random values from the generateArrayLength arb, and the contents of the array provided by the generateContents arb. | ✓ | ✓ | ✓ |
 | **Booleans** |
-| `Arb.boolean()` | Returns random true and false values. | ✓ | ✓ | ✓ |
+| `Arb.bool()` | Returns an `Arb` that produces `Boolean`s. | ✓ | ✓ | ✓ |
```

Comment:
Was this renamed to boolean ?

Reply:
No, I will rename it in a future PR. Right now, however, it is named `Arb.bool`. Seems like the documentation has always been wrong here.